server/src/main/java/org/opensearch/index/mapper/IpFieldMapper.java

```diff
@@ -256,7 +256,14 @@ public Query termQuery(Object value, @Nullable QueryShardContext context) {
                 String term = value.toString();
                 if (term.contains("/")) {
                     final Tuple<InetAddress, Integer> cidr = InetAddresses.parseCidr(term);
-                    return InetAddressPoint.newPrefixQuery(name(), cidr.v1(), cidr.v2());
+                    PointRangeQuery pointsRange = (PointRangeQuery) InetAddressPoint.newPrefixQuery(name(), cidr.v1(), cidr.v2());
+                    return SortedSetDocValuesField.newSlowRangeQuery(
```

Comment:
I am wondering if that actually correct? We are basically treating the term query as range query, but it looks like we should have just return:

```
return SortedSetDocValuesField.newSlowExactQuery(name(), new BytesRef(((PointRangeQuery) query).getLowerPoint()));
```